backend/lib/models/schemas/v2/author.js

```diff
@@ -4,16 +4,16 @@ const { schema: locationSchema } = require("./location");
 
 // -- Schema
 const authorSchema = new Schema({
-  authorId: {
+  id: {
     ref: "User",
     required: true,
     type: ObjectId,
   },
-  authorName: {
+  name: {
     required: true,
     type: String,
   },
-  authorType: {
```

Comment:
I find the word `type` is quote ambiguous here and could mean many things, might be confusing or limiting for future changes.
Also because of the merging of "organization" with "indivudual" as  user... since there are 2 different sub types of a user, maybe those should have the types that only apply to a individual or organization user?